### DISCUSSION TITLE
USB product string descriptor?

### DISCUSSION THREAD

**Discussion by grandaspanna** [earlephilhower/arduino-pico#317] [Q&A]
This seems to be defined to "Pico" here: pico-sdk/src/rp2_common/pico_stdio_usb/stdio_usb_descriptors.c

I know you're not supposed change the vendor/product ID's, but is it ok to change the product string?

If so, what's the recommended way of doing this?

**Comment by earlephilhower** [comment]
That file is not used by the Arduino core.  You can change the USB strings and IDs in the boards.txt file if you really want. 

````
rpipico.build.vid=0x2e8a
rpipico.build.pid=0x000a
rpipico.build.usb_manufacturer="Raspberry Pi"
rpipico.build.usb_product="Pico"
````

**Reply by grandaspanna** [reply]
Thanks. Looked like I started my search too far down the tree :-)
